docs/R-IOC/r10_tidyverse.md

```diff
@@ -0,0 +1,222 @@
+## 1 Luke Skywalker    172   77       blond        fair      blue       19.0   male masculine  Tatooine
+## 2          C-3PO    167   75        <NA>        gold    yellow      112.0   none masculine  Tatooine
+## 3          R2-D2     96   32        <NA> white, blue       red       33.0   none masculine     Naboo
+## 4    Darth Vader    202  136        none       white    yellow       41.9   male masculine  Tatooine
+## 5    Leia Organa    150   49       brown       light     brown       19.0 female  feminine  Alderaan
+## 6      Owen Lars    178  120 brown, grey       light      blue       52.0   male masculine  Tatooine
+  species
+## 1   Human
+## 2   Droid
+## 3   Droid
+## 4   Human
+## 5   Human
+## 6   Human
+##                                                                                                                                       films
+## 1                                           The Empire Strikes Back, Revenge of the Sith, Return of the Jedi, A New Hope, The Force Awakens
+## 2                    The Empire Strikes Back, Attack of the Clones, The Phantom Menace, Revenge of the Sith, Return of the Jedi, A New Hope
+## 3 The Empire Strikes Back, Attack of the Clones, The Phantom Menace, Revenge of the Sith, Return of the Jedi, A New Hope, The Force Awakens
+## 4                                                              The Empire Strikes Back, Revenge of the Sith, Return of the Jedi, A New Hope
+## 5                                           The Empire Strikes Back, Revenge of the Sith, Return of the Jedi, A New Hope, The Force Awakens
+## 6                                                                                     Attack of the Clones, Revenge of the Sith, A New Hope
+##                             vehicles                starships
+## 1 Snowspeeder, Imperial Speeder Bike X-wing, Imperial shuttle
+## 2                                                            
+## 3                                                            
+## 4                                             TIE Advanced x1
+## 5              Imperial Speeder Bike                         
+## 6          
+```
+
+!!! abstract "Take a break & Read"
+    For a detailled description of `tibbles`, please go read the [section 10](https://r4ds.had.co.nz/tibbles.html) 
+    of R4DS (O'Reilly Book on tidyverse).
+
+
+### Manipulating data
+
+Some of tidyverse packages such as `dplyr`, `tidyr` and `stringr` are used for manipulating 
+your data. You can add/remove columns or rows, filter or manipulating strings and tables. 
+
+!!! abstract "Take a break & Read"
+    To introduce the tidyverse, you can read the [section 3 Data Transform](https://r4ds.hadley.nz/data-transform) 
+    of R4DS 2e edition where you can be able to see basic commands for manipulating 
+    `data.frames` and `tibbles` thanks to `dplyr`.
+
+#### Pipe operator
+
+In the previous section, you must have notice some weird code : `|>`. It's called a 
```

Comment:
I think we can still mention it's the same as the previous `%>%` operator (in case someone already know the old one).

Reply:
It's explained in the ressource ! And as it's not shown in the example, I'd rather not talk about at this moment.